janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphTest.java

```diff
@@ -5057,6 +5059,8 @@ public void read(Message message) {
                         txNo = 2;
                         //v2 addition transaction
                         assertEquals(1, Iterables.size(changes.getVertices(Change.ADDED)));
+                        String vertexLabel = changes.getVertices(Change.ADDED).iterator().next().label();
+                        assertEquals(testVertexLabel, vertexLabel);
```

Comment:
This two lines would fail without the business logic change because `vertexLabel` here would be equal to `vertex` even so real label of the vertex label is `testVertex`. With this PR this vertex now returns correct vertex label.